### PR TITLE
Remove unused usings

### PR DESCRIPTION
**Pull request by panopticoncentral**
Super-minor cleanup, just removes all the unused "using" statements across the project.

### CODE REVIEW

**Review by Forgind — APPROVED** [review]
I think this is the first time I've seen GitHub not load diffs because there were too many files rather than because a file was too large.

**Comment by rainersigwald** [comment]
@panopticoncentral How did you do this? What tool? Were any manual fixups required?

**Comment by panopticoncentral** [comment]
Just opened in the latest VS, clicked on an unused using and chose "clean up solution." I thought I built retail but guess not! Will fix that up.

If this isn't desirable, it's no big deal to just close, it's just a force of habit for me. :)

**Comment by panopticoncentral** [comment]
Or maybe it wasn't retail, I may have forgotten to build tests.

File: src/Shared/ToolsetElement.cs [hunk 1]
```diff
@@ -306,7 +303,7 @@ public PropertyElementCollection PropertyElements
         /// </summary>
         internal sealed class PropertyElementCollection : ConfigurationElementCollection
         {
-            #region Private Fields
+    #region Private Fields
```
Comment (ladipro) [inline src/Shared/ToolsetElement.cs:306]: It looks like the convention, at least in MSBuild, is to align the `#region` / `#endregion` directives same as the enclosed code, so this change is likely not desired.

File: src/Tasks/FxCopExclusions/Microsoft.Build.Tasks.Suppressions.cs [hunk 2]
```diff
@@ -1,12 +1,4 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-// FxCop Suppression file
-// To Use:
-// Add module level suppressions to this file to have them suppressed in the assembly
-//
```
Comment (ladipro) [inline src/Tasks/FxCopExclusions/Microsoft.Build.Tasks.Suppressions.cs:6]: Same here. Is this being deleted intentionally?

File: src/Utilities/FxCopExclusions/Microsoft.Build.Utilities.Suppressions.cs [hunk 3]
```diff
@@ -1,12 +1,4 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-// FxCop Suppression file
-// To Use:
-// Add module level suppressions to this file to have them suppressed in the assembly
-//
```
Comment (ladipro) [inline src/Utilities/FxCopExclusions/Microsoft.Build.Utilities.Suppressions.cs:6]: Same here. Intentional?

File: src/Utilities/TrackedDependencies/CanonicalTrackedInputFiles.cs [hunk 4]
```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Build.Utilities
     /// </summary>
     public class CanonicalTrackedInputFiles
     {
-        #region Member Data
+#region Member Data
```
Comment (ladipro) [inline src/Utilities/TrackedDependencies/CanonicalTrackedInputFiles.cs:24]: Undesired indentation change?

File: src/Utilities/TrackedDependencies/CanonicalTrackedOutputFiles.cs [hunk 5]
```diff
@@ -19,25 +19,25 @@ namespace Microsoft.Build.Utilities
     /// </summary>
     public class CanonicalTrackedOutputFiles
     {
-        #region Member Data
+#region Member Data
```
Comment (ladipro) [inline src/Utilities/TrackedDependencies/CanonicalTrackedOutputFiles.cs:22]: Undesired indentation change?

File: src/Utilities/TrackedDependencies/DependencyTableCache.cs [hunk 6]
```diff
@@ -28,7 +28,7 @@ internal static class DependencyTableCache
         /// </summary>
         internal static Dictionary<string, DependencyTableCacheEntry> DependencyTable { get; } = new Dictionary<string, DependencyTableCacheEntry>(StringComparer.OrdinalIgnoreCase);
 
-        #region Methods
+#region Methods
```
Comment (ladipro) [inline src/Utilities/TrackedDependencies/DependencyTableCache.cs:31]: Undesired indentation change?

File: src/Utilities/TrackedDependencies/FileTracker.cs [hunk 7]
```diff
@@ -58,7 +65,7 @@ public enum ExecutableType
     /// </summary>
     public static class FileTracker
     {
-        #region Static Member Data
+#region Static Member Data
```
Comment (ladipro) [inline src/Utilities/TrackedDependencies/FileTracker.cs:68]: Undesired indentation change?

File: src/Utilities/TrackedDependencies/FlatTrackingData.cs [hunk 8]
```diff
@@ -21,12 +21,12 @@ namespace Microsoft.Build.Utilities
     /// </summary>
     public class FlatTrackingData
     {
-        #region Constants
+#region Constants
```
Comment (ladipro) [inline src/Utilities/TrackedDependencies/FlatTrackingData.cs:24]: Undesired indentation change?

**Comment by drewnoakes** [comment]
This code fix shouldn't be changing any other code. I'd file a bug against Roslyn for these.

**Review by ladipro — APPROVED** [review]

**Review by Forgind — APPROVED** [review]

File: src/Deprecated/Engine/Solution/VCProjectParser.cs [hunk 9]
```diff
@@ -3,12 +3,11 @@
 
 using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Xml;
 
 namespace Microsoft.Build.BuildEngine
 {
-	static internal class VCProjectParser
+    static internal class VCProjectParser
```
Comment (rainersigwald) [inline src/Deprecated/Engine/Solution/VCProjectParser.cs:10]: Revert whitespace change.

**Review by rainersigwald — CHANGES_REQUESTED** [review]
@Forgind volunteered to get merge conflicts resolved.

**Review by rainersigwald — APPROVED** [review]

**Comment by rainersigwald** [comment]
Thanks @panopticoncentral!

**Pull request merged**